env.sh

```diff
@@ -99,7 +99,7 @@ EOF
   run_cmd cd "$cmssw_base/src"
   run_cmd eval `scramv1 runtime -sh`
   if ! [ -d "CombineHarvester" ]; then
-    run_cmd bash <(curl -s https://raw.githubusercontent.com/cms-analysis/CombineHarvester/main/CombineTools/scripts/sparse-checkout-ssh.sh)
+    run_cmd git clone https://github.com/cms-analysis/CombineHarvester.git CombineHarvester
```

Comment:
Please, do not include env modification in this PR, and create a separate PR instead.